src/Elastic.Transport/Requests/RequestParameters.cs

```diff
@@ -2,28 +2,36 @@
 // Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information
 
+using System;
 using System.Collections.Generic;
+using System.Collections.Specialized;
+using Elastic.Transport.Extensions;
 using static Elastic.Transport.UrlFormatter;
 
 // ReSharper disable once CheckNamespace
 namespace Elastic.Transport;
 
 /// <summary>
-/// 
+///
 /// </summary>
 public interface IStringable
 {
 	/// <summary>
-	/// 
+	///
 	/// </summary>
 	/// <returns></returns>
 	string GetString();
 }
 
 /// <summary>
-/// 
+///
 /// </summary>
-public sealed class DefaultRequestParameters : RequestParameters
+public sealed class DefaultRequestParameters : RequestParameters;
+
+/// <summary>
+///
+/// </summary>
+public static class RequestParameterExtensions
 {
```

Comment:
Can we remove this empty `RequestParameterExtensions` class or does it have a hidden purpose?